plugins/pipelines/syncPRs.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.pipeline.base, 'id');
+
+module.exports = () => ({
+    method: 'GET',
```

Comment:
I think it could be either a GET or a PUT. But since it's been `GET` for `sync` and `syncWebhooks`, I keep it the same to be consistent.